packages/data/addon/adapters/elide-metadata.ts

```diff
@@ -38,21 +39,22 @@ export default class GraphQLMetadataAdapter extends EmberObject {
    * @param {Object} options
    * @returns {Object} JSON response for type
    */
-  async fetchMetadata(type: MetadataQueryType, id: string | undefined, options: MetadataRequestOptions = {}) {
+  fetchMetadata(type: MetadataQueryType, id: string | undefined, options: MetadataRequestOptions = {}) {
     const query: DocumentNode | undefined = GQLQueries[type]?.[id ? 'single' : 'all'];
 
     assert(
       'Type requested in navi-data/elide-metadata adapter must be defined as a query in the gql/metadata-queries.js file',
       query
     );
 
-    const queryOptions = Object.assign({}, options, { query });
-    if (id) {
-      queryOptions.variables = Object.assign(queryOptions.variables || {}, {
-        ids: [id]
-      });
-    }
+    const queryOptions = {
+      query,
+      variables: {
+        ...(isPresent(id) && { ids: [id] })
+      },
+      ...options
```

Comment:
`options` is of type MetadataRequestOptions which also has a `variables` key so we could see some collision there